Doc/library/random.rst

```diff
@@ -412,6 +412,31 @@ Alternative Generator
       ``None``, :class:`int`, :class:`float`, :class:`str`,
       :class:`bytes`, or :class:`bytearray`.
 
+   .. method:: Random.seed()
```

Comment:
Is this signature correct? The docs for `random.seed()` give `random.seed(a=None, version=2)` https://docs.python.org/3/library/random.html#random.seed